readers/influxdb/messages_test.go

```diff
@@ -498,7 +498,7 @@ func TestReadJSON(t *testing.T) {
 
 	for desc, tc := range cases {
 		result, err := reader.ReadAll(tc.chanID, tc.pageMeta)
-
+		
```

Comment:
Please remove this blank line

Reply:
Error handling is missing here,
should be is this wanted or should ´require.NoError´

Reply:
I meant the empty (blank) line that you added below.